doc/charm++/manual.rst

```diff
@@ -9715,8 +9865,8 @@ above can be used for this.
 
 .. _lbWriteNewLB:
 
-Writing a new load balancing strategy
-~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
+Writing a new load balancing strategy (Legacy centralized load balancer)
```

Comment:
Just to clarify - is writing a new strategy in the old way still possible?

Reply:
Also, does the section starting on line 9916 apply to the old and/or new way?

Reply:
In any case, I think we might consider removing the legacy documentation.

Reply:
Yes, writing a strategy in the old way is still possible. The section starting on 9916 applies to only the old way, the new one doesn't involving changing linking at all, it just adds another strategy to the set of the ones that TreeLB can plug into its structure.

It should certainly be cleaned up, the trouble is that both the old and the new way have their purposes for now: the new for anything that can fit into the structure of TreeLB, the old for legacy support and for strategies that are unsuitable for the current TreeLB (they use the comm graph, are distributed, etc.).